app/public/js/movie.js

```diff
@@ -1,4 +1,4 @@
-var decision_time = 15;
+var decision_time = 12;
```

Comment:
It's not all! You have to set the timer interval [here](https://github.com/tino-junge/cinema-app/blob/master/app/public/css/application.css#L63). Now the timer (the animation) doesn't time out quickly enough.